content/docs/rfcs/37/README.md

```diff
@@ -0,0 +1,151 @@
+This effectively emulates a mechanism to _"re-activate"_ a stale Noise session by binding it to a newly created active Noise session.
+
+In order to reduce users' metadata leakage, it is desirable (as suggested in [35/WAKU2-NOISE](https://rfc.vac.dev/spec/35/#after-handshake)) that content topics used for communications change every time a new message is exchanged.
+This can be easily realized by employing a key derivation function to compute a new `session-id` from the previously employed one (e.g. `session-id = HKDF(prev-session-id)`),
+while keeping the Inbound/outbound Cipher States, the content topic derivation mechanism and the stale mechanism the same as above.
+In this case, when one party sends **and** receives at least one message, 
+he SHALL publicly mark as stale all Noise sessions relative to messages exchanged before the earlier of these two send/receive events.
+
+# Multi-Device support
+
+Alice and Bob might possess one or more devices (e.g. laptops, smartphones, etc.) they wish to use to communicate.
+In the following we assume Alice and Bob to possess $N$ and $M$ devices, respectively.
+
+Since a Noise session contains cryptographic material required to encrypt and decrypt messages exchanged on a pre-defined content topic derived from a `session-id`, 
+messages should be encrypted and decrypted within the Noise session instantiated between the currently-in-use sender's and receiver's device.
+
+This is achieved through two main supported session management mechanisms that we called `N11M` and `NM`, respectively.
+
+## The $N11M$ session management mechanism
+
+In a $N11M$ setting, each party's device shares the same Noise session information used to encrypt and decrypt messages exchanged with the other party.
+
+![](./N11M.png)
+
+More precisely, once the first Noise session between any of Alice's and Bob's device is instantiated, 
+its session information is securely propagated to all other devices, 
+which then become able to send and receive new messages on the content topic associated to such session.
+
+The most updated session information between Alice and Bob is propagated in encrypted form to other devices, 
+using previously instantiated Noise sessions. 
+The propagated session information corresponds to the latest session information stored on the device currently communicating with (any of the devices of) the other party.
+
+We note that sessions information is propagated only among devices belonging to the same party and not with other party's devices. 
+Hence, Alice has no knowledge on the number of devices Bob is using and vice versa. 
+
+When any device marks a Noise session between Alice and Bob as stale, 
+all (updated) devices will implicitly mark such session as stale.
+
+In case a Noise session between two devices belonging to the same party is marked as stale, 
```

Comment:
Have you explained that devices of the same person instantiate a Noise session?

Reply:
Yes, it is few lines earlier: https://github.com/vacp2p/rfc/blob/2cb76227edc8bfd061999a7ae42c89fecbb2b0a5/content/docs/rfcs/37/README.md?plain=1#L97-L98

Reply:
>  instantiated Noise sessions.  
Can you please explain when does/did the `previously Noise sessions` happen?

Reply:
Please see this comment https://github.com/vacp2p/rfc/pull/504#discussion_r880971264

Reply:
Addressed in https://github.com/vacp2p/rfc/commit/f4c6b40ed3647120770a6330355c09e854d45e0c